core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -27,7 +27,7 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 
 /**
- *
+ * Table metadata result when loading a table
```

Comment:
I believe @rdblue is in the middle of following up here as well, I think in this case I would just drop the empty javadoc in anticipation of a commit which adds a more specific doc.

Reply:
fix via drop the empty javadoc